pkg/reconciler/identity/controller.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package identity
+
+import (
+	"context"
+	kubeclient "knative.dev/pkg/client/injection/kube/client"
+)
+
+func NewIdentity(ctx context.Context) *Identity {
```

Comment:
I would just put this in the same identity/reconciler.go file and remove this file